src/wp-includes/fonts/font-library/class-wp-font-collection.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
```

Comment:
Rather than putting the Font Library class files into its own directory, instead move all of them into the `wp-includes/fonts/` directory. 

Why? [This structure was discussed and agreed to during the introduction of Font Face](https://github.com/WordPress/wordpress-develop/pull/5051#pullrequestreview-1593806620). It better aligns to how Core structures its files.

Reply:
Done https://github.com/WordPress/wordpress-develop/pull/5285/commits/635d8703e9d570c92813a9e69e8b76915f7dc88a